process/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG - process
 
+## 1.5.0 / 2018-10-12
+
+* [Added] Upgrade psutil. See [#2190](https://github.com/DataDog/integrations-core/pull/2190).
```

Comment:
same, I don't think we should release this.

Reply:
Why not?

Reply:
I think this got attached to the wrong code line. I'm fine with releasing this 👍 (And think we should) This was supposed to be for one of the checks where the ONLY changelog entry was "add command to validate metadata".